internal/plugin/grpc_client_plugin_test.go

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) F5, Inc.
+//
+// This source code is licensed under the Apache License, Version 2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+
+package plugin
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/nginx/agent/v3/internal/bus"
+	"github.com/nginx/agent/v3/internal/config"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func getAgentConfig() *config.Config {
+	return &config.Config{
```

Comment:
Linter should be complaining about this too
```suggestion
func getAgentConfig(t *testing.T) *config.Config {
        t.Helper()
	return &config.Config{
```

Same applies for the other `getAgentConfig()` test helper functions that have been added.

Reply:
I'll do a follow up PR to centrailize this stuff I think. There's config everywhere so it would be better to have it in one place